lib/scarlet/plugin.rb

```diff
@@ -23,10 +24,18 @@ def helper(*args, &block)
   end
 
   class Context
-    def initialize *objs
+    attr_accessor :event
+
+    def initialize event, helpers, *objs
+      @event = event
+      extend(helpers)
```

Comment:
rather:

```ruby
# inside Scarlet::Plugin
def initialize(...)
  cxt = Scarlet::Context.dup.extend(helpers)
end

def run
  cxt.new()
end
```

Simply storing the context on plugin init (note that helpers will already be defined by then, since `helper` is a class method), then using it in place instead of `Scarlet::Context`